### ISSUE TITLE
Trying to get ValuesListQuerySet throws "TypeError: _clone() takes exactly 1 argument (2 given)"

### ISSUE THREAD

**Issue by jkkjonah** [makinacorpus/django-safedelete#90]
I defined an extremely simple SafeDeleteModel and when I try to get a ValuesListQuerySet, the following error is thrown. Any idea why this is happening? Willing to contribute a fix, but could use some guidance.

```
TypeError: _clone() takes exactly 1 argument (2 given)
```

My model:
```
class DumbModel(SafeDeleteModel):
    is_thing = models.BooleanField(default=False)
```

My code:
```
In [1]: from app.models import DumbModel
In [2]: DumbModel.objects.all().values_list()
Out[2]: <repr(<django.db.models.query.ValuesListQuerySet at 0x106df54d0>) failed: TypeError: _clone() takes exactly 1 argument (2 given)>

In [3]: DumbModel.objects.all().values_list()[0]
Out [3]: TypeError: _clone() takes exactly 1 argument (2 given)
```

Django version: 1.8.14
django-safedelete version: 0.4.4

**Comment by jkkjonah** [comment]
Test that reproduces the issue w/ Django 1.8.*:  https://github.com/makinacorpus/django-safedelete/pull/92

**Comment by Gagaro** [comment]
https://github.com/makinacorpus/django-safedelete/commit/2fd3a5f36f8683d575fc74efbd496a8a1b62757e#commitcomment-27160176

> I took a stab a this. I have no clue why this happed. I read the Django code base. Seems like QuerySet takes self, flat and *kwargs as arguments. No clue why this happens: https://github.com/django/django/blob/1.8.18/django/db/models/query.py#L1144

This is what we have for 1.11: https://github.com/django/django/blob/1.11.9/django/db/models/query.py#L1106

And the traceback:

```
Traceback (most recent call last):
  File "/home/travis/build/makinacorpus/django-safedelete/safedelete/tests/test_queryset.py", line 202, in test_values_list
    QuerySetModel.objects.filter(id=instance.id).values_list('pk', flat=True)[0]
  File "/home/travis/build/makinacorpus/django-safedelete/safedelete/queryset.py", line 121, in __getitem__
    return super(SafeDeleteQueryset, self).__getitem__(key)
  File "/home/travis/build/makinacorpus/django-safedelete/django/django/db/models/query.py", line 199, in __getitem__
    qs = self._clone()
  File "/home/travis/build/makinacorpus/django-safedelete/django/django/db/models/query.py", line 1244, in _clone
    clone = super(ValuesListQuerySet, self)._clone(*args, **kwargs)
  File "/home/travis/build/makinacorpus/django-safedelete/django/django/db/models/query.py", line 1144, in _clone
    c = super(ValuesQuerySet, self)._clone(klass, **kwargs)
TypeError: _clone() takes 1 positional argument but 2 were given
```

**Comment by Gagaro** [comment]
Our implementation is incorrect, we overwrite the `_clone` method:

https://github.com/makinacorpus/django-safedelete/blob/master/safedelete/queryset.py#L139-L147

**Comment by Gagaro** [comment]
Fixed in ab96e7b620a1604a95b9229797c786aeb4c3c144, thanks everyone for the help.

**Comment by Gagaro** [comment]
0.4.5 has been released with the fix
